domino-ui/src/main/java/org/dominokit/domino/ui/dropdown/DropdownActionsGroup.java

```diff
@@ -40,7 +40,7 @@ public DropdownActionsGroup(Node titleElement) {
         return create((Node) titleElement);
     }
 
-    public static <T> DropdownActionsGroup<T> create(IsElement titleElement) {
+    public static <T, E extends HTMLElement> DropdownActionsGroup<T> create(IsElement<E> titleElement) {
```

Comment:
we can make this `public static <T> DropdownActionsGroup<T> create(IsElement<? extends HTMLElement> titleElement) {`